metadata-ingestion/src/datahub/ingestion/source/sigma/sigma_api.py

```diff
@@ -22,7 +22,6 @@ def __init__(self, config: SigmaSourceConfig) -> None:
         self.config = config
         self.workspaces: Dict[str, Workspace] = {}
         self.users: Dict[str, str] = {}
```

Comment:
my comment was about users and workspaces too - those should probably not be getting saved here?